yast2_rdp.pm

```diff
@@ -0,0 +1,194 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: this test checks that YaST2's RDP module is behaving
+#          correctly and verifying that RDP service has been
+#          successfully set.
+# Maintainer: Jun Wang <jgwang@suse.com>
+
+use base "y2_module_guitest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use version_utils 'is_sle';
+
+my $repos = {'12-SP1' => 'http://download.suse.de/ibs/Devel:/YaST:/SLE-12-SP1/SLE-12-SP1/',
+    '12-SP1-UP' => 'http://download.suse.de/ibs/Devel:/YaST:/SLE-12-SP1/SLE-12-SP1_Update/',
+    '12-SP2'    => 'http://download.suse.de/ibs/Devel:/YaST:/SLE-12-SP2/SLE_12_SP2/',
+    '12-SP2-UP' => 'http://download.suse.de/ibs/Devel:/YaST:/SLE-12-SP2/SUSE_SLE-12-SP2_Update/',
+    '12-SP3'    => 'http://download.suse.de/ibs/Devel:/YaST:/SLE-12-SP3/SLE_12_SP3/',
+    '12-SP3-UP' => '',
+    '12-SP4'    => 'http://download.suse.de/ibs/Devel:/YaST:/SLE-12-SP4/SLE_12_SP4/',
+    '12-SP4-UP' => 'http://download.suse.de/ibs/Devel:/YaST:/SLE-12-SP4/SUSE_SLE-12-SP4_Update/'
+};
+
+sub becomeroot {
```

Comment:
? , what about `select_console` or http://open.qa/api/testapi/#_become_root ?

Reply:
Actually I had a try "become_root" testapi, but it produced too many reference pictures, this is not good to review the result. So I simplified "become_root" to "becomeroot" for reducing reference pictures.

Reply:
Well, you could improve the existing `become_root` function but defining another one that is similarly named and confusing is most certainly not the way to go

Reply:
> Well, you could improve the existing `become_root` function but defining another one that is similarly named and confusing is most certainly not the way to go

Yes, this point is important, I will care it.